README.md

```diff
@@ -1,41 +1,123 @@
 # News Summary challenge
 
-* Feel free to use Google, your notes, books, etc. but work on your own.
-* If you refer to the solution of another coach or student, please put a link to that in your README.
-* If you have a partial solution, **still check in a partial solution** and send in a pull request.
-* You must submit a pull request to this repo with your code by 9am Monday morning.
+A News Summary app:
+- Written in JavaSript
+- No libraries or frameworks - just pure js
+- Uses APIs from the Guardian and Aylien to GET the articles
+- Designed for Politicians using these [user stories](#user-stories)
 
-## Challenge
+## Getting Started
 
-As usual please start by forking this repo.
+To use the app, checkout the deployed version here:
 
-You'll create an app that summarises the news.
+[The News](http://rebel-ocean.surge.sh/)
```

Comment:
I don't know if these are problems with the hosting, but there were a few glitches for me when running this in the browser. Some of the links to go to the article summaries don't work, and some of them do. I also had one headline showing twice at one point but that may well have been a glitch in the API

Reply:
yep definitely is glitchy, I would really like to find out why! Ran out of steam on Sunday to track down the reason